bots/leo-monitor/index.js

```diff
@@ -1,160 +1,156 @@
 "use strict";
 var leo = require("leo-sdk");
 var AWS = require("aws-sdk");
-var https = require('https');
-var querystring = require('querystring');
 
 const ID = "leo_cron_monitor";
 
 exports.handler = function(event, context, callback) {
-	refreshCredentials().then(() => {
-		var loader = leo.load(ID, "monitor");
-		event.Records.forEach((record) => {
-			let now = record.dynamodb.ApproximateCreationDateTime * 1000;
-			var newImage = {
-				trigger: 0,
-				invokeTime: 0
-			};
-			var oldImage = {
-				trigger: 0,
-				invokeTime: 0
-			};
-			if ("NewImage" in record.dynamodb) {
-				newImage = AWS.DynamoDB.Converter.unmarshall(record.dynamodb.NewImage);
-			}
-			if ("OldImage" in record.dynamodb) {
-				oldImage = AWS.DynamoDB.Converter.unmarshall(record.dynamodb.OldImage);
-			}
+	// refreshCredentials().then(() => {
```

Comment:
This was strange. If you look below this "refreshCredentials" function always resolves...